test/parallel/test-cli-node-print-help.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+const common = require('../common');
+
+// The following tests assert that the node.cc PrintHelp() function
+// returns the proper set of cli options when invoked
+
+const assert = require('assert');
+const { exec } = require('child_process');
+let stdOut;
+
+
+function startPrintHelpTest() {
+  exec('./node --help', common.mustCall((err, stdout, stderr) => {
+    if (err) {
+      assert.ifError(err);
```

Comment:
This doesn't need to be wrapped in `if (err) {}`.

Reply:
done